src/main/java/camp/woowak/lab/customer/service/SignUpCustomerService.java

```diff
@@ -31,19 +33,19 @@ public SignUpCustomerService(CustomerRepository customerRepository, PayAccountRe
 	 * @throws InvalidCreationException 구매자 생성에 오류가 나는 경우
 	 * @throws DuplicateEmailException 이메일이 중복되는 경우
 	 */
-	@Transactional
-	public Long signUp(SignUpCustomerCommand cmd) {
+	@Transactional()
```

Comment:
옵션을 넣지 않았는데 괄호가 추가되어있습니다!
```suggestion
	@Transactional
```
옵션이 없으면 괄호가 없는것이 좋을 것 같습니다!

Reply:
앗 동훤님이 한발 빨랐네요

Reply:
허걱걱 [9b89988](https://github.com/woowa-techcamp-2024/Team1-3K1K/pull/58/commits/9b899885919d64c1b644aaceb9527157590eec19) 에서 반영했습니다.